draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -939,30 +939,17 @@ following conditions holds:
 
 ## "Same-site" and "cross-site" Requests  {#same-site-requests}
 
-Two origins, A and B, are considered same-site if the following algorithm
-returns true:
+Two origins are same-site if they satisfy the same-site criteria defined in
```

Comment:
This seems to be defined in html, not fetch (and there it's spelled "same site" without a hyphen). https://html.spec.whatwg.org/multipage/origin.html#same-site

Reply:
Good catch! I've updated the text here to fix both the reference and spelling issue.